content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -8,23 +8,17 @@ weight: 70
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
 The TTL controller provides a TTL (time to live) mechanism to limit the lifetime of resource
```

Comment:
For GA, I would _prefer_ to rename this to “The TTL-after-finished controller”

for example:
```suggestion
The TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides
a TTL (time to live) mechanism to limit the lifetime of Kubernetes workload
```